README.md

```diff
@@ -64,14 +64,25 @@ In local environment, the plugin will not work properly because you will not be
 
 6. Copy migrations and templates
     ```shell
-    cp -R vendor/sylius/refund-plugin/src/Migrations/* src/Migrations
     cp -R vendor/payplug/sylius-payplug-plugin/src/Migrations/* src/Migrations
-    sed -i -- 's/Sylius\\RefundPlugin\\Migrations/DoctrineMigrations/g' src/Migrations/*
     bin/console doctrine:migrations:migrate
     mkdir -p templates/bundles/SyliusAdminBundle/
     cp -R vendor/payplug/sylius-payplug-plugin/src/Resources/views/SyliusAdminBundle/* templates/bundles/SyliusAdminBundle/
     ```
 
+    :grey_exclamation: Only for **Sylius 1.7**
+    ```shell
+    cp -R vendor/sylius/refund-plugin/src/Migrations/* src/Migrations
+    sed -i -- 's/Sylius\\RefundPlugin\\Migrations/DoctrineMigrations/g' src/Migrations/*
+    ```
+
+    :grey_exclamation: Only for **Sylius 1.8** and more, update `config/packages/doctrine_migrations.yaml` by add
```

Comment:
IMHO, the :grey_exclamation: is too inconspicuous. 

May I suggest the use of :warning:  ? 
```suggestion
    :warning: Only for **Sylius 1.8** and more, update `config/packages/doctrine_migrations.yaml` by adding following config
```